docs/react-native/v2/features/join.mdx

```diff
@@ -3,98 +3,63 @@ title: Join Room
 nav: 3.1
 ---
 
-Joining a room is one of the most basic functionalities so we at 100ms have built an easy, developer friendly API to get you started.
+## Overview
 
-A user can interact with participants of a room only after joining the room.
+To join and interact with others in audio or video call, the user needs to join a `room`.
 
-## Setting up the HMS Instance
+When user indicates that they want to _join_ the room, your app should have -
 
-First, create an instance of HMSManager class. Store this instance as a property. We'll have to call build method, that method returns an instance of HMSManager class and the same is used to perform all the operations. Ensure that the SDK object is alive in memory so that you can receive event callbacks from SDK. Simplest way to do this is as follows -
+1. User Name - The name which should be displayed to other peers in the room.
+2. [Authentication Token](../guides/token) - The Client side Authentication Token generated by the Token Service.
 
-```js
-import { HMSSDK } from '@100mslive/react-native-hms';
-
-const hmsInstance = await HMSSDK.build();
-```
-
-## Check Permissions
+You can also optionally pass these fields -
 
-Now you can proceed to request Camera and Audio permissions if already not given.
+1. Settings - Such as to join room with Muted Audio or Video using the `HMSTrackSettings` object.
+2. User Metadata - This can be used to pass any additional metadata associated with the user using `metadata` of `HMSConfig` object.
```

Comment:
```suggestion
2. User Metadata - This can be used to pass any additional metadata associated with the user using `metadata` property of `HMSConfig` object.
```